### PR TITLE
new file should be added

### PR DESCRIPTION
**Pull request by Arnab-Developer**
Related to #8

### CODE REVIEW

File: new-file.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+this is a new file.
```
Comment (arnab-jit1) [inline new-file.txt:1]:
```suggestion
This is a new file.
```

File: new-file.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+This is a new file which has been created to test PR review process.
```
Comment (arnab-jit1) [inline new-file.txt:1]: Give a heading

**Review by arnab-jit1 — APPROVED** [review]

File: new-file.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
+---- New file ----
```
Comment (Arnab-Developer) [inline new-file.txt:1]: Can I update the title?

Comment (arnab-jit1) [reply new-file.txt:1]: Ok

Comment (arnab-jit1) [reply new-file.txt]: further update the heading

**Review by arnab-jit1 — APPROVED** [review]

**Pull request merged**